test/e2e/metrics_e2e_test.go

```diff
@@ -80,6 +97,18 @@ func getOLMPod(client operatorclient.ClientInterface) *corev1.Pod {
 	return &podList.Items[0]
 }
 
+func getCatalogPod(client operatorclient.ClientInterface) *corev1.Pod {
```

Comment:
This function seems to be extremely specific to a single test. It should probably be made more generic (takes a label) or just implemented inline.

Reply:
@awgreene good catch. Updated the PR with a generic function.